app/scripts/components/FormStatusModal/FormStatusModal.tsx

```diff
@@ -71,50 +71,42 @@ const FormStatusModal = ({
         >
           &times;
         </button>
-        <h4 translate="yes">Form Status</h4>
+        <h4>Form Status</h4>
```

Comment:
I assume you wanted to take the `translate` attribute off these elements, but I just want to double-check.

Reply:
Correct. It doesn't do anything, we aren't doing label translation in this project, and it was causing a distracting JSX error for each occurrence.